src/components/badge/badge.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component, h, Prop } from '@stencil/core';
+const ONE_MILION = 1000000;
+const ONE_THOUSAND = 1000;
+const FOUR_DIGIT_LENGTH = 4;
+const FIVE_DIGIT_LENGTH = 5;
+
+@Component({
+    tag: 'limel-badge',
+    styleUrl: 'badge.scss',
+    shadow: true,
+})
+export class Badge {
+    /**
+     * Shows numbers in the badge
+     */
+    @Prop()
+    public badgeCounter: number;
+
+    showRoundedNumber() {
+        const digitLength = this.badgeCounter.toString().length;
+        if (digitLength < FOUR_DIGIT_LENGTH) {
+            return this.badgeCounter;
+        }
+        const getParam = this.filter(digitLength);
+        let badgeCounter = (this.badgeCounter / getParam.devisor)
+            .toFixed(1)
+            .toString();
+
+        // to remove the last 0 digit and the dot. Ex 10.0k will be 10k
+        if (badgeCounter.substring(badgeCounter.length - 1) === '0') {
+            const numbersToRemove = -2;
+            badgeCounter = badgeCounter.slice(0, numbersToRemove);
+        }
+        return `${badgeCounter}${getParam.postfix}`;
+    }
+
+    filter(digitLength: number) {
+        let devisor;
+        let postfix;
+        if (digitLength > FIVE_DIGIT_LENGTH) {
+            devisor = ONE_MILION;
```

Comment:
Just some misspelling. "divisor" and "ONE_MILLION"

Reply:
I removed this code.